libs/features/spectrum-web-components/dist/icons-workflow.js

```diff
@@ -2,6 +2,7 @@
 /* Generated by Milo */
 
 import{html as B}from"./base.js";import{IconBase as Z}from"./icon.js";var g,a=function(t,...e){return g?g(t,...e):e.reduce((r,o,L)=>r+o+t[L+1],t[0])},i=t=>{g=t};var u=({width:t=24,height:e=24,hidden:r=!1,title:o="Alert"}={})=>a`<svg
+    aria-hidden="true"
```

Comment:
Changes in this file can be reverted, attributes exist.

Reply:
Reverted 👍